src/video-player.js

```diff
@@ -520,6 +522,9 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     if (publicId instanceof VideoSource) {
       return this.videojs.cloudinary.source(publicId, options);
     }
+    if (this.videojs.shoppable) {
+      this.videojs.shoppable(this.videojs, options.shoppable);
+    }
```

Comment:
@yaniv-li should this be wrapped with a build condition? in `/src/plugins/index.js` you wrap it.

Reply:
it can be but it not really required as this checks if the shoppable plugin is loaded